### PR TITLE
darkspawn get trolled less from incendiary rounds

### PR DESCRIPTION
**Pull request by TheRyeGuyWhoWillNowDie**
who is the clown ass who created this glorified rock paper scissors gamemode

# Changelog

:cl:  
tweak: darkspawn heatmod down to 1.25 from 1.5
tweak: lightward sigil now removes fire stacks over time
/:cl:

### CODE REVIEW

**Comment by LoliconSlayer** [comment]
i ded

File: yogstation/code/modules/mob/living/carbon/human/species_types/darkspawn.dm [hunk 1]
```diff
@@ -64,6 +64,8 @@
 			H.SetSleeping(0)
 			H.setOrganLoss(ORGAN_SLOT_BRAIN,0)
 			H.setCloneLoss(0)
+		if(upgrades["light_resistance"])
+			H.fire_stacks--
```
Comment (Moltijoe) [inline yogstation/code/modules/mob/living/carbon/human/species_types/darkspawn.dm]: please use adjust_fire_stacks

**Review by Moltijoe — CHANGES_REQUESTED** [review]

File: yogstation/code/modules/mob/living/carbon/human/species_types/darkspawn.dm (hunk 1 above)
Comment (TheRyeGuyWhoWillNowDie) [reply yogstation/code/modules/mob/living/carbon/human/species_types/darkspawn.dm]: ☝🤓

**Comment by Simplehorror** [comment]
ok how do i kill them now

File: yogstation/code/modules/mob/living/carbon/human/species_types/darkspawn.dm (hunk 1 above)
Comment (Simplehorror) [reply yogstation/code/modules/mob/living/carbon/human/species_types/darkspawn.dm]: lmfao

**Comment by TheRyeGuyWhoWillNowDie** [comment]
> ok how do i kill them now

attack them until their hit points fall to 0 while keeping your hit points above 0

**Pull request closed without merging**